### PR TITLE
describe the abcd archive connection

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[VAT-Docs: 2p](https://trello.com/c/TDjNe4ri/1763-vat-docs-2p)

File: src/features/abcd.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+The data is displayed on the map as clustered points and can then be used like any other data in the VAT system.
+Zooming in will dissolve the clusters and show the individual occurrences.
+Users can also open the data table to see more attributes of the occurrences.
+
+## Multimedia items
+
+![Multimedia items](media/abcd-spiders-2.png)
+
+Some ABCD datasets contain linkts to multimedia items.
```
Comment (dbrandenstein) [inline src/features/abcd.md]: links

File: src/features/abcd.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+These can be images, videos, or audio files.
+Users can click on the multimedia item in the data table to open it in a new dialog.
+For instance, when the item is an image, it will be displayed directly in the VAT system.
+
+The data table will show at most 3 links for clustered occurrences.
```
Comment (dbrandenstein) [inline src/features/abcd.md]: three

**Review by dbrandenstein — CHANGES_REQUESTED** [review]

File: src/features/abcd.md (hunk 2 above)
Comment (ChristianBeilschmidt) [reply src/features/abcd.md]: [a8bc18d](https://github.com/umr-dbs/vat-docs/pull/11/commits/a8bc18d4efa5839ccf6113038bd1e8d5681892de)

File: src/features/abcd.md (hunk 1 above)
Comment (ChristianBeilschmidt) [reply src/features/abcd.md]: [a8bc18d](https://github.com/umr-dbs/vat-docs/pull/11/commits/a8bc18d4efa5839ccf6113038bd1e8d5681892de)

**Review by dbrandenstein — APPROVED** [review]

**Pull request merged**